docker-compose-esignet/config/esignet-default.properties

```diff
@@ -48,7 +48,7 @@ spring.messages.encoding=UTF-8
 spring.main.allow-bean-definition-overriding=true
 
 ## ------------------------------------------------- e-Signet ----------------------------------------------------------
-mosip.esignet.amr-acr-mapping-file-url=https://raw.githubusercontent.com/mosip/mosip-config/collab/amr-acr-mapping.json
+mosip.esignet.amr-acr-mapping-file-url=https://raw.githubusercontent.com/mosip/mosip-config/release-1.4.1-ES/amr-acr-mapping.json
```

Comment:
We should point to the tag . Please update this to below
https://raw.githubusercontent.com/mosip/mosip-config/v1.4.0-ES/amr-acr-mapping.json